### PR TITLE
RCAL-325: Updating Photom DataModels in RDM

### PR DESCRIPTION
**Pull request by PaulHuwe**
Updating photom maker utilities and tests.

### CODE REVIEW

File: src/roman_datamodels/testing/factories.py [hunk 1]
```diff
@@ -663,18 +664,50 @@ def create_wfi_img_photom_ref(**kwargs):
     roman_datamodels.stnode.WfiImgPhotomRef
     """
     raw_dict = {
+        "F062":
+            {"photmjsr": (10 * np.random.random() * u.MJ / u.sr),
```
Comment (ddavis-stsci) [inline src/roman_datamodels/testing/factories.py:668]: Can this be something like,
"photmjsr": (1.0e-15 * np.random.random() * u.MJ / u.sr
and
"uncertainty": 1.0e-15 * np.random.random() * u.MJ / u.sr
To get the magnitude more or less correct.

File: src/roman_datamodels/testing/factories.py [hunk 2]
```diff
@@ -663,18 +664,50 @@ def create_wfi_img_photom_ref(**kwargs):
...
+             "uncertainty": np.random.random() * u.MJ / u.sr,
+             "pixelareasr": .2 * u.sr},
```
Comment (ddavis-stsci) [inline src/roman_datamodels/testing/factories.py:670]: This is a pixel area of 0.2 sr? I think that something like
"pixelareasr":  1.0e-13 * u.sr}
might be closer to the final values.

**Review by ddavis-stsci — APPROVED** [review]
The one thing I'd update are the default values. If I'm reading this correctly the default values would be 1-10 for photmjsr, 0-1 for uncertainty and the pixel area is 0.2 steradians. It might be better to have more realistic values. I've put some suggestions below.

File: src/roman_datamodels/testing/factories.py (hunk 1 above)
Comment (PaulHuwe) [reply src/roman_datamodels/testing/factories.py:668]: Updated - also updated the maker utility to match.

**Pull request merged**